gpflow/likelihoods/robustmax.py

```diff
@@ -16,11 +16,18 @@ class RobustMax(tf.Module):
 
     with
 
-    y_i = (1-eps)  i == argmax(f)
-          eps/(k-1)  otherwise.
+    y_i = (1-epsilon)  i == argmax(f)
```

Comment:
I'm surprised that you didn't use unicode here. You can also add to the docs that Ɛ is ``epsilon`` property.

```suggestion
    y_i = (1 - Ɛ)  i == argmax(f)
```

Reply:
I didn't use unicode because the parameter is also called `epsilon` not `Ɛ` (I'd be open to a discussion about the pros and cons of unicode-variable names in code though :wink: )